src/cascadia/TerminalSettingsModel/ActionMap.cpp

```diff
@@ -683,18 +683,20 @@ namespace winrt::Microsoft::Terminal::Settings::Model::implementation
     // - keys: the key chord of the command to search for
     // Return Value:
     // - the command with the given key chord
-    // - nullptr if the key chord is explicitly unbound
+    // - nullptr if the key chord doesn't exist
     Model::Command ActionMap::GetActionByKeyChord(Control::KeyChord const& keys) const
     {
-        // Check the current layer
-        const auto cmd{ _GetActionByKeyChordInternal(keys) };
-        if (cmd.has_value())
+        const auto modifiers = keys.Modifiers();
+
+        if (auto vkey = keys.Vkey())
         {
-            return *cmd;
+            if (auto command = _GetActionByKeyChordInternal({ modifiers, vkey, 0 }))
+            {
+                return *command;
+            }
         }
 
-        // This key chord is not explicitly bound
-        return nullptr;
+        return _GetActionByKeyChordInternal({ modifiers, 0, keys.ScanCode() }).value_or(nullptr);

```

Comment:
We'll now have to run `_GetActionByKeyChordInternal` twice, since the given `keys` might have both a vkey as well as a scan code set, but the internal hashmaps have only key bindings loaded that contain either of the two.

I used to have a comment explaining this here, but I must've lost it at some point. I'll re-add one.